src/docker/endpoint-registry/Dockerfile

```diff
@@ -24,6 +24,24 @@ RUN cd /pkg/aas-logger \
     && yarn run build
 
 
+
+## aas-logger ##
+COPY pkg/aas-logger/package.json /pkg/aas-logger/
+COPY pkg/aas-logger/package-lock.json /pkg/aas-logger/
+RUN cd /pkg/aas-logger \
+    && npm install --production\
+    && mkdir /aas-logger-production-dependencies/ \
+    && cp -R node_modules /aas-logger-production-dependencies/ \
+    && npm install
+
+
+COPY pkg/aas-logger/ /pkg/aas-logger/
+
+RUN cd /pkg/aas-logger \
+    && npm run clean \
+    && npm run build
+
+
```

Comment:
Why add this part at all? 
That whole part is already build with yarn in the same image! 
It crashes the docker build.

Reply:
This looks like a merge error. the yarn block is the correct one.

Reply:
I will revert the Dockerfile to the canary version. The changes to the file were faulty introduced